public/app/features/search/components/ManageDashboards.tsx

```diff
@@ -153,7 +153,9 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
       display: flex;
       flex-direction: column;
       flex: 1;
-      margin-top: ${theme.spacing.xl};
+      height: 100%;
+      padding-top: ${theme.spacing.lg};
+      margin-top: -${theme.spacing.lg};
     `,
```

Comment:
this should not be needed, just set the parent wrapper to be:

display: flex
flex-direction: column;
height: 100%;

And then the child you want to take the remaining space (left after toolbar):
flex: 1 1 0;
min-height: 0;

This should

Reply:
Thanks, that worked, except `min-height: 0;` was breaking the layout, so removing it fixed the issue.